extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -64,6 +64,10 @@ amp-sidebar[side] {
   animation-fill-mode: forwards;
 }
 
+amp-sidebar[style] {
```

Comment:
Shouldn't this apply to sidebars even if they don't have the `style` attribute?

Reply:
Yeah good point. I moved it directly as a style declaration under amp-sidebar instead.